python/cuml/thirdparty_adapters/adapters.py

```diff
@@ -295,16 +295,17 @@ def _masked_column_median(arr, masked_value):
     mask = _get_mask(arr, masked_value)
     if arr.size == 0:
         return cp.full(arr.shape[1], cp.nan)
-    arr_sorted = arr.copy()
+    arr_sorted = arr
```

Comment:
```suggestion
```
This definition is not necessary; `arr_sorted` is given the definition it actually uses in both branches below.